examples/tutorials/flow-deploy/kubernetes/README.md

```diff
@@ -0,0 +1,154 @@
+
+Then run the command below:
+
+```shell
+cd build
+docker build . -t web-classification-serve
+```
+
+### Create Kubernetes deployment yaml.
+The Kubernetes deployment yaml file acts as a guide for managing your docker container in a Kubernetes pod. It clearly specifies important information like the container image, port configurations, environment variables, and various settings. Below, you'll find a simple deployment template that you can easily customize to meet your needs.
+
+You need encode the secret using base64 firstly and input the encoded value as 'open-ai-connection-api-key' in the deployment configuration, for example:
+```shell
+encoded_secret=$(echo -n <your_api_key> | base64)
+```
+```yaml
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: <your-namespace>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: open-ai-connection-api-secret
+  namespace: <your-namespace>
+type: Opaque
+data:
+  open-ai-connection-api-key: <encoded_secret>
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: web-classification-service
+  namespace: <your-namespace>
+spec:
+  type: NodePort
+  ports:
+  - name: http
+    port: 8080
+    targetPort: 8080
+    nodePort: 30123
+  selector:
+    app: web-classification-serve-app
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: web-classification-serve-app
+  namespace: <your-namespace>
+spec:
+  selector:
+    matchLabels:
+      app: web-classification-serve-app
+  template:
+    metadata:
+      labels:
+        app: web-classification-serve-app
+    spec:
+      containers:
+      - name: web-classification-serve-container
+        image: <your-docker-image>
+        imagePullPolicy: Never
+        ports:
+        - containerPort: 8080
+        env:
+        - name: OPEN_AI_CONNECTION_API_KEY
+          valueFrom:
+            secretKeyRef:
+              name: open-ai-connection-api-secret
+              key: open-ai-connection-api-key
+```
+
+### Apply the deployment.
+Before you can deploy your application, ensure that you have set up a Kubernetes cluster and installed [kubectl](https://kubernetes.io/docs/reference/kubectl/) if it's not already installed. In this documentation, we will use [Minikube](https://minikube.sigs.k8s.io/docs/) as an example. To start the cluster, execute the following command:
+```shell
+minikube start
+```
+Once your Kubernetes cluster is up and running, you can proceed to deploy your application by using the following command:
+```shell
+kubectl apply -f deployment.yaml
+```
+This command will create the necessary pods to run your application within the cluster.
+
+### Retrieve flow service logs of the container
+The kubectl logs command is used to retrieve the logs of a container running within a pod, which can be useful for debugging, monitoring, and troubleshooting applications deployed in a Kubernetes cluster.
+
+```shell
+kubectl -n <your-namespace> logs <pod-name>
```

Comment:
we can use real value in sample

Reply:
updated some value which can be fixed.

Pod names are typically generated by Kubernetes using a combination of the deployment's name and a unique identifier, so they are not directly user-configurable when creating a deployment.

such as web-classification-serve-app-**67cd7446fd-snbwc**